src/FlatButton/FlatButton.js

```diff
@@ -93,6 +93,15 @@ class FlatButton extends Component {
      * Override the inline-styles of the root element.
      */
     style: PropTypes.object,
+    /**
+     * Target attribute applied to the button.
+     */
+    target: PropTypes.oneOf([
```

Comment:
Hi, thank you for your feedback. I do agree that it should be `string`, I've reverted the modifications and only updated the propTypes for documentation purposes.